aws-api/src/main/java/com/amplifyframework/api/aws/AWSApiPlugin.java

```diff
@@ -781,7 +781,7 @@ public Builder apiAuthProviders(@NonNull ApiAuthProviders apiAuthProviders) {
         /**
          * Apply customizations to an underlying OkHttpClient that will be used
          * for a particular API.
-         * @param forApiName The name of the API for which these customizations should apply
+         * @param forApiName The name of the API for which these customizations should apply.  This can be found in your `amplifyconfiguration.json` file.
```

Comment:
```suggestion
         * @param forApiName The name of the API for which these customizations should apply.
                             This can be found in your `amplifyconfiguration.json` file.
```